pynemo/nemo_bdy_extr_tm3.py

```diff
@@ -87,10 +87,13 @@ def __init__(self, setup, SourceCoord, DstCoord, Grid, var_nam, grd, pair):
         self.var_nam = var_nam
         sc_z = SC.zt[:]
         sc_z_len = len(sc_z)
-        
+      
         
         self.jpj, self.jpi = DC.lonlat[grd]['lon'].shape
         self.jpk = DC.depths[grd]['bdy_z'].shape[0]
+        self.bdy_dz= DC.depths[grd]['bdy_dz']
+        if grd=='t':
+            self.bdy_msk=DC.bdy_msk
```

Comment:
in the move to py3 the write-out of bdy_msk go lost. Popped it into the Extract object to carry over to the write subroutine